src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -783,9 +783,14 @@ TerminalInput::OutputType Terminal::SendCharEvent(const wchar_t ch, const WORD s
 // Return Value:
 // - none
 TerminalInput::OutputType Terminal::FocusChanged(const bool focused)
-{
-    return _getTerminalInput().HandleFocus(focused);
-}
+   {
+       if (_isFocused != focused)
+       {
+           _isFocused = focused;
+           return _getTerminalInput().HandleFocus(focused);
+       }
+       return {};
+   }

```

Comment:
It looks like there are some whitespace formatting issues here that the Code Health checker will probably complain about. There's a `runformat.cmd` script in the `terminal/tools` directory that should fix stuff like this for you automatically.